src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -108,6 +110,10 @@ public function load(array $configs, ContainerBuilder $container)
             ->addTag('api_platform.doctrine.orm.query_extension.item');
         $container->registerForAutoconfiguration(QueryCollectionExtensionInterface::class)
             ->addTag('api_platform.doctrine.orm.query_extension.collection');
+        $container->registerForAutoconfiguration(MongoDbQueryItemExtensionInterface::class)
```

Comment:
those one should depend on persistence layer decission.

Reply:
Why?